modules/apps/object/object-web/src/main/resources/META-INF/resources/js/tests/ObjectField/FormulaContainer.spec.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {render, screen} from '@testing-library/react';
+import React from 'react';
+
+import '@testing-library/jest-dom/extend-expect';
+
+import {FormulaContainer} from '../../components/ObjectField/Tabs/BasicInfo/FormulaContainer';
```

Comment:
Hey @DavysonMelo, I could verify that the ordering pattern for this type of import is:

```
import '@testing-library/jest-dom/extend-expect';
import {render, screen} from '@testing-library/react';
import React from 'react';

import {FormulaContainer} from '../../components/ObjectField/Tabs/BasicInfo/FormulaContainer';
```

I figured this would be pointed out by the source format, sorry for not requesting this change earlier.